src/implementations/twig/components/gallery/gallery.html.twig

```diff
@@ -80,6 +82,10 @@
 {% if _grid %}
   {% set _css_class = _css_class ~ ' ecl-gallery--grid' %}
 
+  {% if _grid_template is not empty %}
+    {% set _css_class = _css_class ~ ' ecl-gallery--template-' ~ _grid_template %}
+  {% endif %}
+
   {% if _column is not empty %}
```

Comment:
this selector is irrelevant when using a "grid template" right? If i'm correct better not setting in those cases and also in the story, if possible, would be less confusing if we would hide the control for the number of columns

Reply:
In fact it is used only when using the "default" grid template (a plain grid without any modification). I'll add a test for that in the twig template, and check the story